src/processing/dom/base-dom-adapter.ts

```diff
@@ -0,0 +1,39 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+
+export default abstract class BaseDomAdapter {
+    EVENTS: string[] = ['onblur', 'onchange', 'onclick', 'oncontextmenu', 'oncopy', 'oncut',
+        'ondblclick', 'onerror', 'onfocus', 'onfocusin', 'onfocusout', 'onhashchange', 'onkeydown',
+        'onkeypress', 'onkeyup', 'onload', 'onmousedown', 'onmouseenter', 'onmouseleave',
+        'onmousemove', 'onmouseout', 'onmouseover', 'onmouseup', 'onmousewheel', 'onpaste', 'onreset',
+        'onresize', 'onscroll', 'onselect', 'onsubmit', 'ontextinput', 'onunload', 'onwheel',
+        'onpointerdown', 'onpointerup', 'onpointercancel', 'onpointermove', 'onpointerover', 'onpointerout',
+        'onpointerenter', 'onpointerleave', 'ongotpointercapture', 'onlostpointercapture',
+        'onmspointerdown', 'onmspointerup', 'onmspointercancel', 'onmspointermove', 'onmspointerover',
+        'onmspointerout', 'onmspointerenter', 'onmspointerleave', 'onmsgotpointercapture', 'onmslostpointercapture'
+    ];
+
+    abstract removeAttr (el: any, attr: string): void;
```

Comment:
FYI: there is a built-in HTMLElement type in typescript

Reply:
https://github.com/DevExpress/testcafe-hammerhead/pull/1876#issuecomment-453951423